### PR TITLE
Even Test

### PR DESCRIPTION
**Pull request by abidina**

### CODE REVIEW

**Comment by csbarrett1** [comment]
👍

**Comment by csbarrett1** [comment]
@katerebekah

File: Mathmagician.Tests/EvenTests.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            Assert.AreEqual(2, ResultingEven);
+        }
+
+        [TestMethod]
+        public void NextEven()
+        {
+            //Arrange
+            Even my_even = new Even();
+            //Act
+            int NextEven = my_even.FollowingEven(4);
```
Comment (katerebekah) [inline Mathmagician.Tests/EvenTests.cs:39]: What is this method FollowingEven supposed to do?

File: Mathmagician.Tests/EvenTests.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        public void EvenSequenceOfNumbers()
+        {
+            //Arrange
+            Even my_even = new Even();
+            //Act
+            List<int> EvenNumberSequence = my_even.EvenSequence(6);
+            List<int> EvenExpectedSequence = new List<int> { 2, 4, 6, 8, 10, 12 };
+            //Assert
+            Assert.AreEqual(EvenExpectedSequence.Count, EvenNumberSequence.Count);
+        }
```
Comment (katerebekah) [inline Mathmagician.Tests/EvenTests.cs:55]: Do a CollectionAssert.AreEqual for your two lists/sequences.

File: Mathmagician.Tests/IntegerTests.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        [TestMethod]
+        public void IntegerSequenceOfNumbers()
+        {
+            //Arrange
+            Integer my_int = new Integer();
+            //Act
+            List<int> NumberSequence = my_int.Sequence(6);
+            List<int> ExpectedSequence = new List<int> {1, 2, 3, 4, 5, 6};
+            //Assert
+            Assert.AreEqual(ExpectedSequence.Count, NumberSequence.Count);
```
Comment (katerebekah) [inline Mathmagician.Tests/IntegerTests.cs:54]: Add a CollectionAssert.AreEqual for the lists/sequences

**Comment by katerebekah** [comment]
Tests look good once you add that last one to both parts

**Comment by katerebekah** [comment]
Those tests look great! :+1:

**Pull request merged**